as2/server/redemption.md

```diff
@@ -0,0 +1,15 @@
+# Redemption
+##### Messages pertaining to the redemption server.
+---
+### Join Server
+Sent to the penguin trying to log in to the redemption server to give needed player information. `rjs`
+
+|Argument|Type|Variadic?|Default
```

Comment:
A "default" column, while somewhat useful, is unncessary as there are no defaults to message arguments in a strict implementation. Should be removed for now.